packages/shadcn/src/components/ui/select.tsx

```diff
@@ -0,0 +1,144 @@
+'use client';
+
+import * as React from 'react';
+import * as SelectPrimitive from '@radix-ui/react-select';
```

Comment:
Rather than use this, import each of the things you need directly from `react` and `@radix-ui/react-select`